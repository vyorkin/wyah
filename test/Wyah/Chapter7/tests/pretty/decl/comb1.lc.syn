Decl "B" (ELam (Var "z") (ELam (Var "y") (ELam (Var "x") (EApp (EVar (Var "x")) (EApp (EVar (Var "y")) (EVar (Var "z")))))))
