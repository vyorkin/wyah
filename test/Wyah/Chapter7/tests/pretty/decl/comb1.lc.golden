let B = \z -> \y -> \x -> x (y z);